runtime/doc/options.txt

```diff
@@ -5287,7 +5287,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 						*'shell'* *'sh'* *E91*
 'shell' 'sh'		string	(default $SHELL or "sh",
-					MS-DOS and Win32: "command.com" or
+					Windows: "command.com" or
```

Comment:
Yes there are lies about `command.com` and `%COMSPEC%` too.

But we can safely change `command.com` to `cmd.exe` I think in the docs.